src/providers/codeactions.ts

```diff
@@ -1,6 +1,7 @@
 import * as vs from 'vscode'
 
 import type { Extension } from '../main'
+import { TeXMathEnvFinder } from '../providers/preview/mathpreviewlib/texmathenvfinder'
```

Comment:
We should not directly import `mathpreviewlib/texmathenvfinder`.

Please make `MathPreview.texMathEnvFinder` public and use `Extension.mathPreview.texMathEnvFinder`.

https://github.com/James-Yu/LaTeX-Workshop/blob/e3c795752b10d0f6dd56df9280002b16bad8b57f/src/providers/preview/mathpreview.ts#L19-L29

Reply:
I have made the necessary changes. For the sake of coherence, I have move all the `replaceXXX` functions inside the `CodeActions` class.